src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -130,6 +130,11 @@ define(function (require, exports, module) {
         case "BRAMBLE_DISABLE_WORD_WRAP":
             PreferencesManager.set("wordWrap", false);
             break;
+        case "BRAMBLE_CONFIGURE_AUTO_CLOSE_TAGS":
+            if(typeof args[0] === "object") {
```

Comment:
Some remote functions pass single arguments, some pass multiple, so we use an array to make it all work.

@cgsingh you can probably dump this `typeof` check and just use `args[0]`, which is what all the other code in here does.  Basically, just do what's on line 135 now, and get rid of the `if` altogether.